NEWS.md

```diff
@@ -1,3 +1,47 @@
+### v9.0.0 (2022-08-03)
+
+* Added official parity support for Node 18.
+
+* Updated unsupported odd-numbered release from 17.x to 19.x to prepare for Node 18 support.
```

Comment:
I don't think we necessarily need to iterate this one out. Its a normal part of the Node 18 activities... but feel free to leave. I might just move below everything else.